composer/datasets/in_context_learning_evaluation.py

```diff
@@ -1040,10 +1042,11 @@ def collate_fn(self, data):
             'test_inputs': test_inputs,  # list of test inputs
             'test_outputs': test_outputs,  # list of test outputs
             'languages': languages,  # list of languages
+            'pass_at_k': self.pass_at_k,
             'generation_length': self.max_seq_len - self.max_prompt_length,
             'generation_kwargs': {
                 'pad_token_id': self.pad_tok_id,
-                'num_beams': self.generations_per_sample,  # change strategy to beam search
+                'num_beams': 1,  # single beam
```

Comment:
does `self.generatations_per_sample` need to be >= `self.pass_at_k`? If so, please add an error if its not.

Reply:
This is a good one. Was debating this myself. Fixed.